deep_speech_2/data_utils/data.py

```diff
@@ -107,9 +113,10 @@ def process_utterance(self, filename, transcript):
         else:
             speech_segment = SpeechSegment.from_file(filename, transcript)
         self._augmentation_pipeline.transform_audio(speech_segment)
-        specgram, text_ids = self._speech_featurizer.featurize(speech_segment)
+        specgram, transcript_part = self._speech_featurizer.featurize(
+            speech_segment, self._keep_transcription_text)
         specgram = self._normalizer.apply(specgram)
-        return specgram, text_ids
+        return specgram, transcript_part
```

Comment:
Please modify the function doc in Line 106.

Reply:
Done.